### PR TITLE
feat: add chunk split guide doc

### PR DESCRIPTION
**Pull request by sanyuan0704**
# PR Details

增加拆包指南文档。

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have updated changeset
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: website/builder/src/zh/guide/advanced/split-chunk.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+      }
+    }
+  }
+}
+```
+
+其中 `override` 中的配置会和 webpack 的配置进行合并，具体配置项请参考 [webpack 官方文档](https://webpack.js.org/plugins/split-chunks-plugin/#splitchunkschunks)。
+
+::: warning 注意
+由于历史原因，当 strategy 为 `custom` 时，需要使用 `splitChunks` 来自定义 webpack 拆包配置，而不是 `override`。
```
Comment (chenjiahan) [inline website/builder/src/zh/guide/advanced/split-chunk.md:174]: 这个历史问题，我们是不是可以处理掉，文档里统一用 override，内部兼容一下 splitChunks 这个用法

File: website/builder/src/zh/guide/advanced/split-chunk.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
+# 拆包最佳实践
+
+良好的拆包策略对于提升应用的加载性能是十分重要的，可以充分利用浏览器的缓存机制，减少请求数量，加快页面加载速度。
+
+在 Builder 中内置了多种拆包策略，可以满足大部分应用的需求，你也可以根据自己的业务场景，自定义拆包配置。
+
+## 拆包策略
+
+Builder 中包括如下的拆包策略：
```
Comment (chenjiahan) [inline website/builder/src/zh/guide/advanced/split-chunk.md:9]: 建议这里给一个 `performance.chunkSplit` 配置项的链接，引导一下用户

File: website/builder/src/zh/guide/advanced/split-chunk.md (hunk 1 above)
Comment (sanyuan0704) [reply website/builder/src/zh/guide/advanced/split-chunk.md:174]: 这里是为了兼容 Eden 配置，特意和 Eden 原来的配置保持一致的。如果要处理掉的话也行，我再去更新下迁移工具

Comment (chenjiahan) [reply website/builder/src/zh/guide/advanced/split-chunk.md:177]: 嗯嗯，我建议文档推荐用统一的写法（override），然后在 Builder 内部兼容一下 splitChunks 这个字段，加个 `@deprecated` 标记，成本不高。

就不单独放到迁移工具里处理了，迁移的话会比较复杂。

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**